### DISCUSSION TITLE
Release v1.12.3 / 5.67.3

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#3467] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

### Release Notes

This build brings a lot of usability improvements most notably the ability to auto force all removable media (**Requires a supporter certificate**).
![grafik](https://github.com/sandboxie-plus/Sandboxie/assets/3890945/0cd7f87c-57a6-4f46-b604-7f5afaa4bfdc)
In the settings exceptions can be specified, based on the volume serial number to exclude selected devices form forced sandboxing.

This build also enhances on the global hot keys, two new hot keys have been added "Alt + Break" to bring the sandman window in front with the top most flag set, and "Ctrl + Alt + F" to toggle disabling of forced processes, furthermore the terminate all (panic hotkey) hot key "Shift + Break" has been improved, individual sandboxes can be configured to be excluded from a blanket global terminate all command, however when the panic hotkey is invoked 3 times with < 1 sec between presses it will terminate all boxed processes, no exceptions.

Also worth mentioning is an improvement to the service handling which allows to install and run the GOG launcher sand boxed in a reduced isolation box with the following configuration:
```
UnrestrictedSCM=y
RunServicesAsSystem=y
NoSecurityIsolation=y
Template=RpcPortBindingsExt
```
Further work is ongoing to make the GOG work in a standard sandbox.

 

For a full list of changes and fixes please review the full [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1123--5673---2023-12-02).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.12.3'>Release v1.12.3 / 5.67.3</a>.</em>

**Comment by BilleBarrett** [comment]
Please tell me how to fix this, getting this error almost daily from clients

![1A-sandboxiemessage](https://github.com/sandboxie-plus/Sandboxie/assets/41338917/065f4365-5161-44f0-953c-6ebe58016119)

**Reply by isaak654** [reply]
Where does UrlInspectAddon.exe come from? Additionally, the notification messages in the second window are cut off.
